### PR TITLE
apache-commons-vfs2: Add package version 2.2

### PR DESCRIPTION
**Pull request by Sumynwa**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Add package version 2.2

###### Change Log  
- Use javapackages-local-bootstrap to avoid build cycle.
- Fix linebreak in sed command.
- Disable hadoop/ftp/ssh fs layer builds.
- Enable check section. Disable tests having dependency on hadoop/ftp/ssh vfs.
- Lint spec
- Add package entry in cgmanifest and licenses

###### Does this affect the toolchain?  
NO

###### Test Methodology

- Pipeline build id: Buddy Build https://dev.azure.com/mariner-org/mariner/_build/results?buildId=270268&view=results

### CODE REVIEW

**Comment by Sumynwa** [comment]
Requesting review on the PR, the apache-commons-* packages have a total of 8 PR's which have dependency on one another. 6 have already been approved.

**Review by mfrw — APPROVED** [review]
LGTM

**Review by oliviacrain — APPROVED** [review]
LGTM once the merge conflict is resolved!

**Pull request merged**
